packages/manager/modules/models/support/level/PartnerLevel.class.js

```diff
@@ -0,0 +1,22 @@
+export default class PartnerLevel {
+  /**
+   *
+   * @param {PartnerLevel} Object payload representing a partner level
+   * @param {PartnerLevel.level} String partnership level
```

Comment:
Maybe you could explain what a level is 

Also, it looks like it's not a string, it's a value of an enum